src/axom/inlet/tests/inlet_Inlet.cpp

```diff
@@ -75,11 +83,11 @@ TEST(inlet_Inlet_basic, getTopLevelBools)
   EXPECT_EQ(table.type(), InletType::Nothing);
 }
 
-TEST(inlet_Inlet_basic, getNestedBools)
+TYPED_TEST(BasicInletTest, getNestedBools)
```

Comment:
Does this get prefixed with anything?  It is a convention in Axom to prefix all tests with their component and filename for filtering purposes.